docker/lr-pb/remove_kinetics.py

```diff
@@ -0,0 +1,37 @@
+import argparse
+import gzip
+from collections import OrderedDict
+from math import ceil
+import pysam
+from construct import *
+
+from multiprocessing.pool import ThreadPool
+from functools import partial
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Remove kinetics information', prog='remove_kinetics')
+    parser.add_argument('-p', '--prefix', type=str, default="out", help="Output prefix")
```

Comment:
If we do decide to compute the name of the new bam, this is no longer necessary.